wire-ios-data-model/Source/MLS/Actions/FetchSubgroupAction.swift

```diff
@@ -18,7 +18,7 @@
 
 import Foundation
 
-public enum SubgroupType: String {
+public enum SubgroupType: String, Hashable, Equatable, Codable {
```

Comment:
Note that there's a mixture of terminology in the source code with "subgroup" and "subconversation". We're going to only use "subconversation" but I'll make these sweeping changes in a separate PR.